src/App.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from "react";
```

Comment:
You no longer need to import React from "react". Starting from the release 17 of React, JSX is automatically transformed without using `React.createElement`.
However, other exports like hooks must be imported.